addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3483,6 +3485,29 @@ public Action Timer_MakeBoss(Handle timer, any boss)
 	{
 		DmgTriple[boss]=true;
 	}
+	
+	SelfKnockback[boss]=false;
+	if(KvGetNum(BossKV[Special[boss]], "knockback", -1)>=0)
+	{
+		SelfKnockback[boss]=KvGetNum(BossKV[Special[boss]], "knockback", -1);
+	}
+	else if(KvGetNum(BossKV[Special[boss]], "rocketjump", -1)>=0)
+	{
+		SelfKnockback[boss]=KvGetNum(BossKV[Special[boss]], "rocketjump", -1);
+	}
+	else
+	{
+		SelfKnockback[boss]=false;
+	}
```

Comment:
I believe a better format for this is:
```sourcepawn
	if(KvGetNum(BossKV[Special[boss]], "knockback", -1)>=0)
	{
		SelfKnockback[boss]=KvGetNum(BossKV[Special[boss]], "knockback", -1);
	}
	else
	{
		SelfKnockback[boss]=KvGetNum(BossKV[Special[boss]], "rocketjump", 0);
	}
```
Or just the `SelfKnockback[boss]=false;` above the if statement.